### PR TITLE
Portfolio Desktop Version

### PR DESCRIPTION
**Pull request by Mukaba**
## 📱+💻 Adding Desktop Version

**Description:** This pull request update the previous merged PRs by adding media queries to make the portfolio responsive in desktop screens 💻 with mobile-first functionality in mind, using HTML and CSS with Grid and Flexbox in order to lay out the elements on the page 📐. 
This upgrade includes now:
-  navigation bar with a logo 📸 ("Logo" by default as in the template) and hamburger menu, 
- a section containing information about the portfolio's owner ✍️, and a list of his contacts 📧.
- a section containing works cards (done projects) about the portfolio's owner ✍️, a list 📧.of languages used by project, and a go live button (see project).
- a section containing languages and skills and a short summary about the portfolio's owner ✍️.
- the last section with a contact form that collect users data and send them to my email using the [Formspree service](https://formspree.io/).
- media queries to make both mobile📱 and desktop 💻 look awesome (Desktop: >=768px).

Great regards

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,10 +10,16 @@
...
     <a href="#" class="self-center" id="myLogo">My Logo</a>
-    <nav class="self-center" id="navbar"></nav>
+    <nav class="self-center" id="navbar">
+        <ul class="" id="nav__list">
+            <li class="nav__li"><a href="#myWorks">Portfolio</a></li>
+            <li class="nav__li"><a href="#aboutMySelf">About</a></li>
+            <li class="nav__li"><a href="#contactMe">Contact</a></li>
+        </ul>
+    </nav>
   </header>
```
Comment (bobb-Rob) [inline index.html:20]: I must say that you did very well. it is beautiful. But I see a few things that can help make the projects even more beautiful.

**The toolbar**

- [x] I notice that this section has about `155px` margin or padding on both the left and right. On your project image below, the red mark signifies how much space should be between the left screen edge and your logo. same with the nav links on the right.
- [x] The space between each nav-link is too much on your design. Kindly ensure the space is same with what we have in the Figma template. 👇🏾 

Figma template. 👇🏾 | Your work 👇🏾 
---- | ----
image | image



Your work 
image

Figma 
image

File: index.html [hunk 2]
```diff
@@ -43,10 +49,11 @@ <h1 class="about__title">I’m Mariane <br> Glad to see <br> you!</h1>
...
-  <section class="works">
+  <section class="works" id="myWorks">
     <div class="works__item tonic__project">
         <div class="tonic__header"></div>
-        <h3 class="project__title">Tonic</h3>
+        <div class="tonic__details">
+            <h3 class="project__title">Tonic</h3>
         <div class="project__tags">
             <p>CANOPY</p>
             <div class="project__tags-boulet gray_color"></div>
```
Comment (bobb-Rob) [inline index.html:59]: - [x] I see that the height of your project cards is too small compared to the Figma Design. Please fix this

Figma Design: height `496px` 
image

Your design: height `272px`
image

File: index.html [hunk 3]
```diff
@@ -172,7 +186,18 @@ <h3 class="about__links">LET'S CONNECT</h3>
...
+        <div class="eclips eclips__11"></div>
+        <div class="eclips eclips__b8"></div>
+        <div class="eclips eclips__b3"></div>
+        <div class="eclips eclips__4"></div>
+        <div class="eclips eclips__12"></div>
+        <div class="eclips eclips__15"></div>
+        <div class="rectangle rectangle__7"></div>
+        <div class="rectangle__b6"></div>
+        <div class="rectangle rectange__b4"></div>
+    </div>
```
Comment (bobb-Rob) [inline index.html:200]: - [x] The `Get in touch` button for your project is different from that of the Figma design. Please fix this.

Figma
image

Your work
image

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @Mukaba , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Mukaba) [reply index.html:59]: ✅ Done

File: index.html (hunk 1 above)
Comment (Mukaba) [reply index.html:20]: ✅ Done

File: index.html (hunk 3 above)
Comment (Mukaba) [reply index.html:200]: ✅ Done

**Review by amejid — APPROVED** [review]
Hi @Mukaba, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- Good UI design :heavy_check_mark: 
- Page is responsive :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**